CMakeLists.txt

```diff
@@ -20,6 +20,7 @@ find_package(daqdataformats REQUIRED)
 find_package(detdataformats REQUIRED)
 find_package(detchannelmaps REQUIRED)
 find_package(hdf5libs REQUIRED)
+find_package(readoutlibs REQUIRED)
```

Comment:
Is there an entry in cmake/triggerConfig.cmake.in?

Reply:
Oops, I forgot that. Thanks for catching it. Fixed now